content/utmt/rooms/creating new rooms/design/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Design
+weight: 1
+resources:
+  - name: decorationTiling
+    src: "decorationTiling.png"
+    title: And finally the decoration and background tiles.
+  - name: oobTiling
+    src: "oobTiling.png"
+    title: Then we add the out of bounds tiles.
+  - name: initialTiling
+    src: "initialTiling.png"
+    title: First we start with the tiles the player is meant to interact with.
+---
+
+TODO: having some more screenshots in here would be nice. Also fix some phrasing.
+
+Now we get to actually laying out and "building" your room. For this, there are two main steps; Setting up collision and tiling.
+{{< hint type="note">}}
+It is generally recommended to do tiling first, as UTMT will always put the visuals for collision blocks in the foreground, meaning that doing tiling afterwards is a horrible experience. 
+{{</hint>}}
+To give your room form, you can place tiles from the many different tilesets. Each tile is added individually by right clicking in the tiles section, selecting `add new tile`, and then dragging the tileset that you want to use over from the left into the definition. This will be a slow process. Each unit or block that represents a tile is 16x16 pixels pixels large, and can be selected manually by setting the source position and size or selected automatically by clicking on the tile you want. However the latter option is sometimes not as good for tilesets which aren't limited to blocks. It is helpful to use an image editor to find the coordinates of specific points for the former option. Tiles that are a multiple of 16x16 can be selected by holding `ALT` while dragging over the tileset thumbnail in the tile's properties.
+Take note of the depth variable when adding tiles. Since UTMT does not have a layer system, this value is what decides which tiles are in the foreground and which are in the background, with lower values being shown in front of higher values.
+
+{{<hint type="tip">}}
+Here are some depth values to take note of:
+- Backgrounds: 100
+- Samus: -1
+- Foreground: -100
+- Breakable Blocks: -111/-112
+- Fading Tiles: -120 (Do not work without an `oFadeTiles` object)
+{{</hint>}}
+
+Something you can use to make the tiling process a lot faster is to copy existing tiles by holding alt and dragging them around the room. This will copy everything from that tile, including depth and tileset. This drastically speeds up the tiling process.  
```

Comment:
```suggestion
Something you can use to make the tiling process a lot faster is to duplicate existing tiles by holding `ALT` while click-dragging them in the room preview. This will copy everything from that tile, including `depth` and `tileset`. This drastically speeds up the tiling process, as you then don't have to manually assign the same values to every tile.  
```